colabcode/code.py

```diff
@@ -60,6 +62,7 @@ def _install_extensions():
             subprocess.run(["code-server", "--install-extension", f"{ext}"])
 
     def _start_server(self):
+        conf.get_default().region = self.region
```

Comment:
please pass region in `options` here: `url = ngrok.connect(addr=self.port, options={"bind_tls": True})`

Reply:
I've tried it out but it does not work by doing that. That's why I have used this method as mentioned in pyngrok docs ([https://pyngrok.readthedocs.io/en/latest/#setting-the-region](https://pyngrok.readthedocs.io/en/latest/#setting-the-region)).

Reply:
Hello Sir, Could you please review this? I've tried the change you mention but it doesn't seem to work.